src/statusFunctions.js

```diff
@@ -0,0 +1,18 @@
+import { tasks, saveTasks } from './todoFunctions.js';
+
+// Function to clear all completed tasks
+const clearCompletedTasks = () => {
+  tasks.forEach((task, index) => {
+    if (task.completed) {
+      tasks.splice(index, 1);
+    }
+  });
+
+  tasks.forEach((task, index) => {
+    task.index = index + 1;
+  });
+
+  saveTasks();
+};
```

Comment:
- [ ] The `Clear all completed` button works as expected but there is a requirement to be completed to pass to the next milestone, please 🙏🏽 use the `filter` method 👇🏽 

> Implement a function for the "Clear all completed" button (use filter() method).

Reply:
thank you for this feed back.